includes/settings/tools.php

```diff
@@ -307,7 +307,7 @@ protected function create_tabs() {
 								'label' => esc_html__( 'Update Site Address (URL)', 'elementor' ),
 								'field_args' => [
 									'type' => 'raw_html',
-									'html' => sprintf( '<input type="text" name="from" placeholder="http://old-url.com" class="large-text"><input type="text" name="to" placeholder="http://new-url.com" class="large-text"><button data-nonce="%s" class="button elementor-button-spinner" id="elementor-replace-url-button">%s</button>', wp_create_nonce( 'elementor_replace_url' ), esc_html__( 'Replace URL', 'elementor' ) ),
+									'html' => sprintf( '<input type="text" name="from" placeholder="https://old-url.com" class="large-text"><input type="text" name="to" placeholder="https://new-url.com" class="large-text"><button data-nonce="%s" class="button elementor-button-spinner" id="elementor-replace-url-button">%s</button>', wp_create_nonce( 'elementor_replace_url' ), esc_html__( 'Replace URL', 'elementor' ) ),
 									'desc' => esc_html__( 'Enter your old and new URLs for your WordPress installation, to update all Elementor data (Relevant for domain transfers or move to \'HTTPS\').', 'elementor' ),
```

Comment:
Great idea to suggest https in the placeholder as well.

The content of the placeholder i.e old-url.com and new-url.com is not a good idea from a security point of view. Some users will type this example just to see what it does. Both domains exist and could be purchased by a bad actor in the future to spread malware.

How about:
https://old.example.com & https://new.example.com

or 
https://example.com/old & https://example.com/new